### ISSUE TITLE
Specify price elasticity for each tier

### ISSUE THREAD

**Issue by christophertull** [California-Data-Collaborative/rate-comparison#1]
Add a way to specify price elasticity of demand for each tier.

Populate with reasonable default values.

**Comment by christophertull** [comment]
I think this is a good idea!

**Comment by patwater** [comment]
Good idea to have more granular price elasticity.  There also could be more detail on the tool though maybe that's not a UI information on the tool thing though rather a UX more in depth how to video and user guide documentation.  

"Price elasticity of demand 
(0 means no price response)"
